src/intTest/resources/edifact/pathology_2.edifact.dat

```diff
@@ -38,7 +38,7 @@ FTX+CID+++TIRED ALL THE TIME, LOW Hb'
 FTX+CID+++COELIAC'
 S16+16'
 SEQ++1'
-SPC+TSP+:::VENOUS BLOOD'
+SPC+TSP+T016:920'
```

Comment:
Is `pathology_2.edifact.dat` for `NHS002` or just `NHS003` pathology messages? Just checking if this is the best place to add `NHS002`  examples

Reply:
`pathology_2.edifact.dat` holds an example NHS002 message (incrementally becoming more and more NHS002ey as we play the stories). `pathology_3.edifact.dat` keeps to the NHS003 expectations.

Reply:
Ok, i didn't realise that - thought it was just another example of `NHS003`